sklearnex/covariance/tests/test_incremental_covariance.py

```diff
@@ -141,3 +147,19 @@ def test_sklearnex_fit_on_random_data(
 
     assert_allclose(expected_covariance, result.covariance_, atol=1e-6)
     assert_allclose(expected_means, result.location_, atol=1e-6)
+
+
+# Monkeypatch IncrementalEmpiricalCovariance into relevant sklearn.covariance tests
+@pytest.mark.parametrize(
+    "sklearn_test",
+    [
+        test_covariance,
+        test_EmpiricalCovariance_validates_mahalanobis,
+    ],
+)
+def test_IncrementalEmpiricalCovariance_against_sklearn(monkeypatch, sklearn_test):
```

Comment:
what is monkeypatch here?

Reply:
https://docs.pytest.org/en/latest/how-to/monkeypatch.html